include/proxy-wasm/wasm_vm.h

```diff
@@ -297,6 +297,11 @@ class WasmVm {
   FOR_ALL_WASM_VM_IMPORTS(_REGISTER_CALLBACK)
 #undef _REGISTER_CALLBACK
 
+  /**
+   * Terminate the current execution of wasm vm, the vm fails and shouldn't run after calling
+   * terminate.
```

Comment:
Style: `Terminate execution of this WasmVM. It shouldn't be used after being terminated.`